corehq/apps/users/views/mobile/users.py

```diff
@@ -667,6 +668,28 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class CreateUserWithLocationView(CreateCommCareUserView):
```

Comment:
wondering if you could make this less impactful and copy/pasted by making `location_id` an optional param / hidden value in the form and then recycling the same view.